CHANGELOG.md

```diff
@@ -11,6 +11,13 @@ The following changes have been implemented but not released yet:
 
 ### Bugfix
 
+#### node and browser
+
+- Fix the initialization of `clientAppId` in `ISessionInfo` objects: `clientAppId` property of the `ISessionInfo` interface, although present, was not being set either in the `ISessionInfo` objects returned by
```

Comment:
Can you make sure the line length is roughly consistent with other entries in the changelog?